### PR TITLE
Add manylinux2010 docker image

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM quay.io/pypa/manylinux2010_x86_64:latest
```
Comment (lalo) [inline vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile:1]: who mantains this image?

File: .github/workflows/build_deploy_latest.yml [hunk 2]
```diff
@@ -11,11 +11,12 @@ jobs:
     strategy:
       matrix:
         config:
-        - { image_name: ubuntu1604-build, dockerfile: vowpal_wabbit/ubuntu1604-build.Dockerfile }
-        - { image_name: ubuntu1404-build, dockerfile: vowpal_wabbit/ubuntu1404-build.Dockerfile }
-        - { image_name: centos7_6_1810-build, dockerfile: vowpal_wabbit/centos7_6_1810-build.Dockerfile }
-        - { image_name: rl-ubuntu-1604, dockerfile: reinforcement_learning/ubuntu1604-build.Dockerfile}
-        - { image_name: all-dev-ubuntu1604, dockerfile: development/ubuntu1604-dev.Dockerfile }
+        - { image_name: ubuntu1604-build, dockerfile: vowpal_wabbit/ubuntu1604-build.Dockerfile, context: ./ }
```
Comment (lalo) [inline .github/workflows/build_deploy_latest.yml:14]: nit: I'm guessing the default context is needed and there's no other way to default to this in all the cases except manylinux?

File: vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile (hunk 1 above)
Comment (lalo) [reply vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile:1]: like how trustworthy is it

**Review by lalo — APPROVED** [review]

File: .github/workflows/build_deploy_latest.yml (hunk 2 above)
Comment (jackgerrits) [reply .github/workflows/build_deploy_latest.yml:14]: I was trying to work this out - there's sneaky things we might be able to do but I just opted for the simple solution

File: vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile (hunk 1 above)
Comment (jackgerrits) [reply vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile:1]: This is one of the standard manylinux images, and is an official part of the Python project. I believe it is trustworthy. https://github.com/pypa/manylinux

File: .github/workflows/build_deploy_latest.yml (hunk 2 above)
Comment (lalo) [reply .github/workflows/build_deploy_latest.yml:14]: yea, thats what i suspected

**Pull request merged**